core/src/ancestor_hashes_service.rs

```diff
@@ -1039,7 +1038,6 @@ mod test {
         repairable_dead_slot_pool: HashSet<Slot>,
         request_throttle: Vec<u64>,
         repair_stats: AncestorRepairRequestsStats,
-        _duplicate_slots_reset_receiver: DuplicateSlotsResetReceiver,
```

Comment:
it seems that it's fine to remove this cfg(test) code added here: https://github.com/solana-labs/solana/pull/18912#discussion_r1174814547

Reply:
Yeah this seems fine to remove given that the `AncestorHashesService` does not shut down if `duplicate_slots_reset_sender` send fails